CorsixTH/Src/iso_fs.cpp

```diff
@@ -370,78 +370,78 @@ IsoFilesystem::file_handle_t IsoFilesystem::findFile(const char* sPath) const
     return 0;
 }
 
-uint32_t IsoFilesystem::getFileSize(file_handle_t iFile) const
+uint32_t iso_filesystem::get_file_size(file_handle iFile) const
 {
-    if(iFile <= 0 || static_cast<size_t>(iFile) > m_iNumFiles)
+    if(iFile <= 0 || static_cast<size_t>(iFile) > file_count)
         return 0;
     else
-        return m_pFiles[iFile - 1].iSize;
+        return files[iFile - 1].size;
 }
 
-bool IsoFilesystem::getFileData(file_handle_t iFile, uint8_t *pBuffer)
+bool iso_filesystem::get_file_data(file_handle iFile, uint8_t *pBuffer)
 {
-    if(iFile <= 0 || static_cast<size_t>(iFile) > m_iNumFiles)
+    if(iFile <= 0 || static_cast<size_t>(iFile) > file_count)
```

Comment:
As a general policy, would it make sense to have this 0 vs 1 stuff in the Lua <-> C++ bridge only?

Reply:
Yes